packages/edit-post/src/components/header/writing-menu/index.js

```diff
@@ -37,11 +45,20 @@ function WritingMenu() {
 				scope="core/edit-post"
 				name="fullscreenMode"
 				label={ __( 'Fullscreen mode' ) }
-				info={ __( 'Work without distraction' ) }
+				info={ __( 'Show and hide admin UI' ) }
 				messageActivated={ __( 'Fullscreen mode activated' ) }
 				messageDeactivated={ __( 'Fullscreen mode deactivated' ) }
 				shortcut={ displayShortcut.secondary( 'f' ) }
 			/>
+			<PreferenceToggleMenuItem
```

Comment:
This is just for testing, correct?

Reply:
From what @shaunandrews proposed this menu item seems to be a path forward.